Modules/readme.md

```diff
@@ -4354,19 +4378,56 @@ Examples
       storage_system_password: password
       storage_system_username: username
       state: add_volume
-      remote_copy_group_name: farhan_rcg
+      remote_copy_group_name: test_rcg
       volume_name: demo_volume_2
       admit_volume_targets:
       - target_name: CSSOS-SSA06
         sec_volume_name: demo_volume_2
 
-  - name: Modify Remote Copy Group farhan_rcg
+  - name: admit Remote Copy target
+    hpe3par_remote_copy:
+      storage_system_ip: 10.10.10.1
+      storage_system_password: password
+      storage_system_username: username
+      state: admit_target
+      remote_copy_group_name: test_rcg
+      target_name: CSSOS-SSA04
+      local_remote_volume_pair_list:
+      - sourceVolumeName: demo_volume_1
+        targetVolumeName: demo_volume_1
+      - sourceVolumeName: demo_volume_2
+        targetVolumeName: demo_volume_2
+      target_mode: periodic
+
+  - name: remote copy group status
+    hpe3par_remote_copy:
+      storage_system_ip: 192.168.67.5
+      storage_system_password: 3pardata
+      storage_system_username: 3paradm
+      state: remote_copy_status
+      remote_copy_group_name: test_rcg
+    register: result
+
+  - debug:
+      msg: "{{ result.output.remote_copy_sync_status}}"
```

Comment:
Change it to msg: "{{ result.output.remote_copy_sync_status }}"
Also, we can add a name to the task to state what this task is doing.